### PR TITLE
[logs] document beforeSend API

### PR DESCRIPTION
**Pull request by hdelaby**
## Motivation

Document `beforeSend` API introduced by https://github.com/DataDog/browser-sdk/pull/644

## Changes

Docs

## Testing

---

I have gone over the [contributing](https://github.com/DataDog/browser-sdk/blob/master/CONTRIBUTING.md) documentation.

### CODE REVIEW

**Review by bcaudan — APPROVED** [review]
LGTM

File: packages/logs/README.md [hunk 1]
```diff
@@ -248,6 +248,70 @@ The placeholders in the examples above are described below:
 
 ## Advanced usage
 
+### Scrub sensitive data from your Browser logs
+
+If your Browser logs data contains sensitive information that need redacting, configure the Browser SDK to scrub sensitive sequences by using the `beforeSend` callback when you initialize the Browser Log Collector.
```
Comment (kayayarai) [inline packages/logs/README.md:253]:
```suggestion
If your Browser logs contain sensitive information that needs redacting, configure the Browser SDK to scrub sensitive sequences by using the `beforeSend` callback when you initialize the Browser Log Collector.
```

File: packages/logs/README.md [hunk 2]
```diff
@@ -248,6 +248,70 @@ The placeholders in the examples above are described below:
...
+
+This callback function gives you access to every event collected by the Browser SDK before they get sent to Datadog.
```
Comment (kayayarai) [inline packages/logs/README.md:255]:
```suggestion
The `beforeSend` callback function gives you access to each event collected by the Browser SDK before it is sent to Datadog, and lets you update commonly redacted properties.
```

File: packages/logs/README.md [hunk 3]
```diff
@@ -248,6 +248,70 @@ The placeholders in the examples above are described below:
...
+
+For example, redact email addresses from your web application URLs:
```
Comment (kayayarai) [inline packages/logs/README.md:257]:
```suggestion
For example, to redact email addresses from your web application URLs:
```

File: packages/logs/README.md [hunk 4]
```diff
@@ -248,6 +248,70 @@ The placeholders in the examples above are described below:
...
+
+| Attribute       | Type   | Description                                                                                      |
+| --------------- | ------ | ------------------------------------------------------------------------------------------------ |
+| `view.url`      | String | The URL of the active web page.                                                                  |
+| `view.referrer` | String | The URL of the previous web page from which a link to the currently requested page was followed. |
+| `message`       | String | The content of the log.                                                                          |
+| `error.stack`   | String | The stack trace or complementary information about the error.                                    |
+| `http.url`      | String | The HTTP URL.                                                                                    |
+
+**Note**: The Browser SDK will ignore modifications made to event properties not listed above. Find out about all event properties on the [Browser SDK repository][5].
```
Comment (kayayarai) [inline packages/logs/README.md:313]:
```suggestion
**Note**: The Browser SDK will ignore modifications made to event properties not listed above. For more information about event properties, see the [Browser SDK repository][5].
```

**Review by kayayarai — CHANGES_REQUESTED** [review]

**Comment by BenoitZugmeyer** [comment]
@kayayarai thank you for your review, I applied your suggestions. Could you check it out?

**Review by kayayarai — APPROVED** [review]
👍  for the docs

**Pull request merged**